theseus/embodied/robot/joint.py

```diff
@@ -69,31 +75,61 @@ def dtype(self) -> torch.dtype:
     def axis(self) -> torch.Tensor:
         return self._axis
 
-    def set_parent(self, parent: int):
+    def set_id(self, id: int):
+        self._id = id
+
+    def set_parent(self, parent: Optional[Link]):
```

Comment:
Is `Optional` intended here, if so should it be followed by a `= None`?

Reply:
The parent can be `None` or `Link`, where `None` means the `parent_link` is corrected to a fixed world frame.

Reply:
I will add a comment there.